app/src/widgets/totd-world-countries/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import useTriggerUpdate from "common/hooks/useTriggerUpdate";
+
+import { WidgetProps } from "../index";
+
+export { saga } from "./sagas";
+
+const TotdWorldCountries: React.FC<Props> = ({
+  id,
+  name,
+  capital,
+  countryCode,
+  currency,
+  languages,
+  tld,
+  flag,
+  meta,
+  triggerUpdate,
+}) => {
+  useTriggerUpdate({ id, params: {}, meta, triggerUpdate }, []);
+
+  return (
+    <div>
+      <div className="text-3 w-100 p-2 text-center">
+        <u>
+          <strong>{name}</strong>
```

Comment:
I suggest moving the title to the widget headline to save some space

Reply:
Sounds good! Did you do that already?

Reply:
Not yet - feel free to take it up, otherwise I will today evening :)

Reply:
How can I do that though? Is there a way to pass it up to the headline?

Reply:
It's part of the translations: you can pass any value with `{{value}}`. For example, the twitter widget shows the twitter username in the headline: https://github.com/darekkay/dashboard/blob/935a2b1e23723a510c349a374180da8ad50087cb/app/src/common/translations/en.json#L157